src/itree.cpp

```diff
@@ -16,13 +16,21 @@
 #include <regex>
 #include <stdexcept>
 #include <string>
+#include <variant>
+#include <cassert>
```

Comment:
Alphabetize, please

Reply:
fnmatch should still be separate as a C-language include, yes?

Reply:
Correct. If it's not part of the C/C++ std lib, group it separately.

Reply:
cool